status/status.go

```diff
@@ -120,15 +120,23 @@ func FromProto(s *spb.Status) *Status {
 }
 
 // FromError returns a Status representing err if it was produced from this
-// package, otherwise it returns nil, false.
+// package. Otherwise, it returns a Status with code.Unknown and the original
+// error message.
```

Comment:
How about:  `Otherwise, ok is false and a Status is returned with codes.Unknown and ....`

Reply:
Sounds good.